### PR TITLE
add vial for 2% milk keyboard

### PR DESCRIPTION
**Pull request by Kuznetsov-s-v**
Hello, I've created a port vial for the 2% milk keyboard by spaceman.

### CODE REVIEW

File: keyboards/spaceman/2_milk/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0x3B, 0x6B, 0xA0, 0x29, 0x80, 0x56, 0xED, 0xD1}
```
Comment (lesshonor) [inline keyboards/spaceman/2_milk/keymaps/vial/config.h]: [Duplicate UID](https://github.com/lesshonor/vial-qmk/actions/runs/6268859061/job/17024458099#step:3:385). You'll need to generate a different one.

File: keyboards/spaceman/2_milk/keymaps/vial/vial.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "m2",
```
Comment (lesshonor) [inline keyboards/spaceman/2_milk/keymaps/vial/vial.json:2]:
```suggestion
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by Lenbok** [comment]
Already addressed by #539

**Comment by Lenbok** [comment]
Sorry, my message was probably too brief. This PR is no longer needed and should be closed. Another PR that adds support for this keyboard had already been created, and it was recently merged.

**Pull request closed without merging**